libc/test/src/fenv/exception_flags_test.cpp

```diff
@@ -12,7 +12,7 @@
 #include "src/__support/FPUtil/FEnvImpl.h"
 #include "test/UnitTest/Test.h"
 
-#include <fenv.h>
```

Comment:
This test uses `fexcept_t`, so it still needs `#include <fenv.h>`.  We will replace it in the next step when we add proxy header for `fexcept_t`.